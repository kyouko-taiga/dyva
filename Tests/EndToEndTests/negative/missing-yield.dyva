subscript f(x, y) =
  if x do
    yield y
