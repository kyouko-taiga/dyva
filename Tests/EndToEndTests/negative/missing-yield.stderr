missing-yield.dyva:3.12: error: subscript must yield before returning
    yield y
           ^
